src/content/docs/en/developers/guides/canvas-badge-integration.mdx

```diff
@@ -0,0 +1,229 @@
+        3. **contentType**: `data/json`
+        4. **responseSchema**
+            
+            ```jsx
+            // eligible
+            {
+            	code: 1,
+            	message: "success",
+            	eligibility: true
+            }
+            
+            // not eligible
+            {
+            	code: 0,
+            	message: "why the recipient is not eligible",
+            	eligibility: false
+            }
+            
+            // error
+            // http code is not 200
+            ```
+        
+        ### Claim Badge
+        
+        1. **method**: GET
+        2. **url**: `{{pathname}}/claim?badge={{badgeContractAddress}}&recipient={{walletAddress}}`
+        3. **contentType**: `data/json`
+        4. **responseSchema**
+            
+            ```jsx
+            // success
+            {
+            	code: 1,
+            	message: "success",
+            	tx: ...
+            }
+            
+            // failure
+            {
+            	code: 0,
+            	message: "some msg",
+            }
+            
+            // error
+            // http code is not 200
+            ```
+    </ToggleElement>
+    
+4. **Access Readiness**: Ensure badge contract address implements `defaultTokenURI`, allowing retrieval of default display data (`name, image, description`) via `bytes32(0)` with informative name and description. 
+5. **Set up support**: Prepare a channel for your badge holders can find you for support, get the url to join that channel ready.
+
+## Step 3 `Test`: Sanity Check 
+
+1. Once your badge has been deployed on Scroll, you can auto-check some of these requirements by running `yarn check-badge`.([link to check badge script](https://github.com/scroll-tech/canvas-contracts/blob/master/script/CheckBadge.s.sol))
+    - If your badge minting transaction reverts, we recommend debugging using `cast`:
+
+    ```bash
+    cast run --rpc-url https://rpc.scroll.io [txhash]
+    ```
+
+    This call will simulate the transaction in a local environment and show you the call stack and revert reason.
+    - For `badgeTokenURI` link returned, ensure correct cross-origin configuration
+    <ToggleElement anchor="reference-cors">
+    <div slot="title">Reference: CORS</div>
+
+            ### CORS
+            
+            > Please ensure the provided API supports the cross-origin requirements.
+            > 
+            
+            ### Sepolia
+            
+            - **localhost**: This allows for development and testing directly from your local machine environment.
+            - **Domains containing the substring 'scroll'**: Any domain that includes the substring 'scroll' in its name is allowed to make requests.
+            
+            ### Mainnet
+            
+            - **`scroll.io` domain**: requests originating from **`scroll.io`** are permitted.
+    </ToggleElement>
+2. **Review the information on the badge detail page**: 
```

Comment:
Where is "the badge detail page"?

Reply:
we dont have link here, the section is asking issuer to check all the details they provide is correct on their preview page.